### DISCUSSION TITLE
how to display the values of a CTkCombobox from another button than its dropdown button

### DISCUSSION THREAD

**Discussion by zorrolaotse** [TomSchimansky/CustomTkinter#2374] [Q&A]
HI, 
Is it possible to display the values of a combobox from another button than its dropdown button?
For example action on the button <B> 
displays the values of the <Combobox V>
                                                  value1
                                                  value2
                                                  value3

Thanks

**Comment by dipeshSam** [comment]
@zorrolaotse Yes, you can simply do it using `cget()` method of `CTkComboBox`.

Sample example:

``` Python
import customtkinter as ctk

def on_click():
    values = cbox.cget("values")
    label.configure(text = f"{values}")

app = ctk.CTk()


cbox = ctk.CTkComboBox(app, values=["option 1", "option 2", "option 3"])
cbox.pack()

button = ctk.CTkButton(app, text="Show options", command = on_click)
button.pack()

label = ctk.CTkLabel(app, text="")
label.pack()


app.mainloop()

```

Output:
![Screenshot_2024_0415_113710.jpg](https://github.com/TomSchimansky/CustomTkinter/assets/72375763/287e0d1d-b3f5-42d7-8594-a62e4d6b52f8)


**Regards**.
